src/components/Book.js

```diff
@@ -5,9 +5,11 @@ import { removeBook } from '../redux/books/books';
 // eslint-disable-next-line react/prop-types
 const Book = ({ book }) => {
   // eslint-disable-next-line react/prop-types
-  const { title, author, id } = book;
+  const {
+    // eslint-disable-next-line react/prop-types
+    title, author, category, id,
```

Comment:
- [OPTIONAL] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉